src/documents/views.py

```diff
@@ -362,7 +362,13 @@ def thumb(self, request, pk=None):
                 handle = doc.thumbnail_file
             # TODO: Send ETag information and use that to send new thumbnails
             #  if available
-            return HttpResponse(handle, content_type="image/png")
+            thumbnail_path = doc.thumbnail_path
+            if os.path.splitext(thumbnail_path)[1] == ".webp":
```

Comment:
it seems `thumbnail_path` is only used once in the next line, consider not creating the variable and just inlining `doc.thumbnail_path` in here